.changeset/cool-experts-sort.md

```diff
@@ -0,0 +1,19 @@
+---
+'@astrojs/starlight': minor
+---
+
+Overhauls the built-in localization system which is now powered by the [`i18next`](https://www.i18next.com/) library and available to use anywhere in your documentation website.
+
+See the [“Accessing UI strings”](https://starlight.astro.build/guides/i18n/#using-ui-translations) guide to learn more about how to access built-in UI labels or your own custom strings in your project. Plugin authors can also use the new [`injectTranslations()`](https://starlight.astro.build/reference/plugins/#injecttranslations) helper to add or update translation strings.
```

Comment:
```suggestion
See the [“Using UI translations”](https://starlight.astro.build/guides/i18n/#using-ui-translations) guide to learn more about how to access built-in UI labels or your own custom strings in your project. Plugin authors can also use the new [`injectTranslations()`](https://starlight.astro.build/reference/plugins/#injecttranslations) helper to add or update translation strings.
```

Should we also update the title with the latest changes?

Reply:
Oh definitely. Find and replace comes back to bite me 😁